### PR TITLE
Feature/#9 uploadfile

### PR DESCRIPTION
**Pull request by phphuong0912**
#About

# (Technical changes)

# Relative URL

# UI changes

# TODO

# Others

### CODE REVIEW

File: app/Http/Controllers/FileController.php [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+
+    /**
+     * Store a newly created resource in storage.
+     *
+     * @param  \Illuminate\Http\Request  $request
+     * @return \Illuminate\Http\Response
+     */
+    public function store(FileRequest $request)
+    {
+        dd($request->all());
```
Comment (Kenini1805) [inline app/Http/Controllers/FileController.php]: remove

File: app/Http/Controllers/FileController.php [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+        if($request->hasfile('filename'))
+        {
+            $destinationPath = 'file_upload';
+            $file->move($destinationPath,$file->getClientOriginalName());
```
Comment (Kenini1805) [inline app/Http/Controllers/FileController.php]:
```suggestion
            $file->move($destinationPath, $file->getClientOriginalName());
```

File: app/Models/File.php [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+    /**
+     * The attributes that should be hidden for arrays.
+     *
+     * @var array
+     */
+
+    public function user()
+    {
+        return $this->belongsTo('App\Models\User');
```
Comment (Kenini1805) [inline app/Models/File.php]:
```suggestion
        return $this->belongsTo(User::class);
```

File: resources/views/file/create.blade.php [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+      $("body").on("click",".btn-danger",function(){ 
+          $(this).parents(".control-group").remove();
+      });
+
+    });
+
+</script>
+</body>
+</html>
```
Comment (Kenini1805) [inline resources/views/file/create.blade.php]: indent

**Comment by Kenini1805** [comment]
OK

File: resources/views/file/create.blade.php [hunk 5]
```diff
@@ -0,0 +1,28 @@
+<html lang="en">
+<head>
+  <title>File upload</title>
```
Comment (trinhductoan) [inline resources/views/file/create.blade.php]: indent

**Pull request merged**